db/schema.rb

```diff
@@ -95,4 +97,12 @@
     t.index ["name"], name: "index_teams_on_name", unique: true
   end
 
+  create_table "users", force: :cascade do |t|
```

Comment:
?

Reply:
:( have pushed a migration to drop the `users` table